src/components/Pagination.tsx

```diff
@@ -0,0 +1,132 @@
+import {useState, useEffect, useRef} from 'react';
+import '../styles/Pagination.scss';
+
+type PaginationProps = {
+  totalRecords?: number,
+  recordsLimit?: number,
+  totalPages?: number,
+  onPageChange?: () => {}
```

Comment:
the function should show return type, here it should looks like () => void;

Reply:
It is temporary function. I thought I need it. I will delete.
But I take advice.